docs/docs/webchat/webchat-button-disable.md

```diff
@@ -0,0 +1,89 @@
+---
+id: webchat-button-disable
+title: Enabling and Disabling Buttons
+---
+
+This functionality allows you to disable a button once the user has clicked on it. Thanks to this visual information, the user can better understand the logical flow and navigate more easily.
```

Comment:
Also to force the user to follow an specific flow. (as he won't be able to click that button again)